docs/src/main/sphinx/connector/pulsar.rst

```diff
@@ -0,0 +1,270 @@
+``pulsar.target-num-splits``                          Desired number of split to use at each query
+``pulsar.max-split-message-queue-size``               Size of message buffer for a split
+``pulsar.max-split-entry-queue-size``                 Size of entry buffer for a split
+``pulsar.max-split-queue-cache-size``                 Limit of buffered message and entry size in byte
+``pulsar.stats-provider``                             StatsProvider to collect Pulsar Trino connector stats
+``pulsar.stats-provider-configs``                     Configuration for StatsProvider in key-value format.
+``pulsar.namespace-delimiter-rewrite-enable``         If enable rewrite Pulsar namespace delimiter
+``pulsar.rewrite-namespace-delimiter``                Delimiter used to rewrite Pulsar's default delimiter '/'
+``pulsar.managed-ledger-offload-max-threads``         Maximum number of thread pool size for ledger offloading
+``pulsar.managed-ledger-offload-driver``              Driver to use to offload/read old data to/from long term storage
+``pulsar.offloaders-directory``                       The directory to locate offloaders nar file
+``pulsar.offloader-properties``                       Properties and configurations related to specific offloader implementation
+``pulsar.auth-plugin``                                Authentication plugin used to authenticate to Pulsar cluster
+``pulsar.auth-params``                                Authentication parameter to be used to authenticate to Pulsar cluster
+``pulsar.tls-allow-insecure-connection``              Whether the Pulsar client accept untrusted TLS certificate from broker
+``pulsar.tls-hostname-verification-enable``           Whether to allow hostname verification when client connects to broker over TLS
+``pulsar.tls-trust-cert-file-path``                   Path for the trusted TLS certificate file of Pulsar broker
+``pulsar.bookkeeper-throttle-value``                  Set Bookkeeper request throttle threshold
+``pulsar.bookkeeper-num-io-threads``                  Set number of IO thread
+``pulsar.bookkeeper-num-worker-threads``              Set number of worker thread
+``pulsar.bookkeeper-use-v2-protocol``                 Whether to use Bookkeeper V2 wire protocol
+``pulsar.bookkeeper-explicit-interval``               The interval to check the need for sending an explicit LAC
+``pulsar.managed-ledger-cache-size-MB``               Size for managed ledger entry cache in MB
+``pulsar.managed-ledger-num-worker-threads``          Number of threads to be used for managed ledger tasks dispatching
+``pulsar.managed-ledger-num-scheduler-threads``       Number of threads to be used for managed ledger scheduled tasks
+``pulsar.nar-extraction-directory``                   Directory to use for extraction Nar file
+================================================      =======================================================================
+
+``pulsar.broker-service-url``
```

Comment:
Do we need to repeat this? It is already above.

Reply:
Lets not use the following setup. I would suggest to work all the content below into the table above. Convert it to use the list-table setup though and stay with the 80 char column hard wrap

Reply:
Even better would be to break the table and config section up into multiple sections for example about required config, bookkeeper config, security config and others and have separate tables. Each section could then have more info to explain details for this topic.

Reply:
@mosabua good suggestion! We've updated.